bids/modeling/statsmodels.py

```diff
@@ -4,6 +4,7 @@
 from collections import namedtuple, OrderedDict, Counter, defaultdict
 import itertools
 from functools import reduce
+from graphviz import Digraph
```

Comment:
We can move this import inside `write_graph`.

Reply:
Fixed.